src/main/java/com/jalasoft/webservice/model/IConvert.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.model;
+
+import com.jalasoft.webservice.entitities.ModelFile;
+
+interface IConvert {
+    ModelFile Convert(ModelFile model); //url
+    //String Convert(Criteria cri)
```

Comment:
remove this comment  and above //url comment

Reply:
done